sources/packages/web/src/components/students/applicationTracker/Draft.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <application-status-tracker-banner
+    label="Draft application"
+    content="Your application has not been submitted to StudentAid BC yet. Please complete and submit your application when you are ready."
+    icon="fa:fas fa-pencil-alt"
+    ><template #actions>
```

Comment:
The button action can be implemented in Draft.vue instead of emitting IMO.

Reply:
edit program route needs properties like 
![image](https://user-images.githubusercontent.com/77353155/192898444-46995bb5-e385-4570-9a55-5cbf094ca99c.png)
.and the function is used used in the parent (already an API is called in the parent to revive these details). emit is used by a child to communicate with their parent in vue.